tests/backup/backup_helper.go

```diff
@@ -1181,13 +1247,128 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
 
 	_, err := task.DoRetryWithTimeout(backupSuccessCheck, time.Duration(retryDuration)*time.Minute, time.Duration(retryInterval)*time.Second)
 	if err != nil {
-		return false, err
+		return false, nil, err
 	}
-	return true, nil
+	backupInspectRequest := &api.BackupInspectRequest{
+		Name:  backupName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+	resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+	if err != nil {
+		return false, nil, err
+	}
+	backupStatus := (resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success)
+	log.Infof("Backup [%s] created successfully", backupName)
+	return backupStatus, resp, nil
+}
+
+func ValidateBackup(backupInspectResponse *api.BackupInspectResponse, clusterAppsContexts []*scheduler.Context) (*BackupRestoreContext, error) {
```

Comment:
This should be named `getBackupContext`

Reply:
I see how that would seem to make sense, but in a future PR, **more code** will be added to this, which will do "deeper validation", which actually involves the exact code which is shown here. Other functions which use `CreateBackup` in the future will also use `ValidateBackup` and they'll most likely also have to use the returned context (which was generated as part of validation), so that they can pass it to `CreateRestore` so that the context can be used to not only **validate** the restore, but also generate the `restoreContext`.

With all that in mind, this function, the way it right now, makes sense.